pkg/proc/proc.go

```diff
@@ -147,6 +147,39 @@ func Continue(dbp Process) error {
 				}
 			}
 			return conditionErrors(threads)
+		case curbp.Active && curbp.Tracepoint:
```

Comment:
This changes the behavior of tracepoints in a backwards incompatible way. It also breaks the `trace` command (not `dlv trace`, the one at the `(dlv)` prompt).

Reply:
We may have to do a major version bump in order to fit this in. I want these improvements to land, but we can't sacrifice backwards compat. 

Also, what breaks at the prompt? Just tested and everything seems to be working.

Reply:
> Also, what breaks at the prompt?

you can set a tracepoint anywhere, not just the header of the function and the expected behavior is to just print something when the breakpoint is hit, no automatic return tracepoint.

Reply:
That's true, I'll think on a way to preserve functionality of this while improving the CLI subcommand.